### PR TITLE
Aleksi

### PR DESCRIPTION
**Pull request by aatuss**
Database to master?

destroydb not working

### CODE REVIEW

**Comment by NuuttiH** [comment]
Is this working for everyone else? It fails to create any tables on my end.

**Comment by NuuttiH** [comment]
Creating tables should work now

**Comment by mikkoaf** [comment]
Please run `git rebase -i master` and squash any excess work-in-progress commits.
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by NuuttiH** [comment]
You should add AUTO_INCREMENT back to company_answer_id and school_answer_id

File: includes/destroydb.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	$table_name4 = $wpdb->prefix . 'School_answer';
+
+	$sql1 = "DROP TABLE IF EXISTS $table_name2
+		;";
+	$sql2 =	"DROP TABLE IF EXISTS $table_name3
+		;";
+	$sql3 =	"DROP TABLE IF EXISTS $table_name4
+		;";
+	require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
+	dbDelta( $sql1 );
```
Comment (mikkoaf) [inline includes/destroydb.php]: Use `$wpdb->query()` instead.
Make sure that the queries are presented in the right order.

Comment (mikkoaf) [inline includes/destroydb.php]: Use $wpdb->query() instead.
Make sure that the queries are presented in the right order.

File: pilot-configurator.php [hunk 2]
```diff
@@ -6,55 +6,43 @@
...
 define( 'PILOT_CONFIGURATOR_PLUGIN_VERSION', '0.0.1' );
-
 /**
   * Require plugin files.
   *
   */
-
 require PILOT_CONFIGURATOR_DIR_PATH . 'includes/class-pilot-configurator.php';
 require PILOT_CONFIGURATOR_DIR_PATH . 'includes/admin/class-pilot-configurator-admin.php';
-require PILOT_CONFIGURATOR_DIR_PATH . 'includes/inno-user-roles.php';
```
Comment (mikkoaf) [inline pilot-configurator.php]: Rebase your commit to include the latest changes in master.

**Pull request merged**